openedx/adg/lms/webinars/forms.py

```diff
@@ -19,8 +19,8 @@ class WebinarForm(forms.ModelForm):
         label=_('Add guests by email address'),
         help_text=_('Add comma separated emails e.g. example1@domain.com,example2@domain.com')
     )
-    send_update_emails_to_registrants = forms.BooleanField(
-        required=False, label=_('Send update email to all the users who have registered')
+    send_update_emails = forms.BooleanField(
+        required=False, label=_('Send update email to registrants and to existing co-hosts, panelists, and presenter')
```

Comment:
Can we change `registrants` to `registered users` for an easier and somewhat simple label?